src/sqladmin-fetcher.ts

```diff
@@ -33,6 +34,27 @@ interface RequestBody {
   access_token?: string;
 }
 
+// https://github.com/googleapis/gaxios is the http request library used by
+// google-auth-library and other Cloud SDK libraries, this function will set
+// a standard default configuration that will ensure retry works as expected
+// for internal google-auth-library requests.
+function setupGaxiosConfig() {
+  gaxios.defaults = {
+    retryConfig: {
+      retry: 3,
+      httpMethodsToRetry: ['GET', 'HEAD', 'PUT', 'POST', 'OPTIONS', 'DELETE'],
```

Comment:
should we only configure the ones we use such as `GET` and `POST`?

Reply:
I'm preserving the methods from the default list and adding `POST` to it, given your comment here that intention might not be explicit enough, maybe with the following that intention is more explicit:

```suggestion
      httpMethodsToRetry: ['POST', ...defaultGaxiosConfig],
```

Reply:
your suggestion makes that more clear, work for me. Thanks!